### PR TITLE
Update wattpad.py

### PR DESCRIPTION
**Pull request by yudilee**
Update wattpad crawler

### CODE REVIEW

File: lncrawl/sources/wattpad.py [hunk 1]
```diff
@@ -24,21 +24,22 @@ def read_novel_info(self):
...
+        self.novel_author = soup.select_one(
+            '.author-info__username').get_text()
         logger.info('Novel author: %s', self.novel_author)
 
-        #description = soup.select('h2.description')[0].get_text()
-
-        chapters = soup.select('ul.table-of-contents a')
-        # chapters.reverse()
+        len(soup.select_one('.story-parts').select('ul li a'))
+
```
Comment (Galunid) [inline lncrawl/sources/wattpad.py:40]: Seems to do nothing, I assume you just missed it before submitting

Comment (dipu-bd) [reply lncrawl/sources/wattpad.py:40]: @yudilee please remove this line before I can merge it

Comment (yudilee) [reply lncrawl/sources/wattpad.py:39]: @Galunid @dipu-bd sorry, my bad forgot to clean up. I updated it . Thank you

Comment (Galunid) [reply lncrawl/sources/wattpad.py:39]: @yudilee Unless I'm missing something that `len(soup.select_one(...))` still does nothing and should be removed

**Pull request merged**